### DISCUSSION TITLE
QED pair production table in epoch

### DISCUSSION THREAD

**Discussion by Aurora-do** [epochpic/epoch#686] [Q&A] [answered]
What is the significance of the three columns in pair production table present in physics packages in epoch? The second and third columns have equal values up to a certain row and then there exist major deviations between the two.

**Comment by Status-Mirror** [comment, marked as answer]
Hi @Aurora-do,

Are you referring to the table file `pairprod.table`? This is one of a few table files used in the pair-production routines, and the values refer to pre-calculated variables from theory. The first few lines of the table reads:

```
1301 -5 8
-5. -50 -50
-4.99 -50 -50
-4.98 -50 -50
-4.97 -50 -50
```

Here, 1301 refers to the number of data-points, and -5 and 8 are ignored. The following rows contain the actual table data.

The first column either represents $\log_{10}(\chi)$ or $\log_{10}(\eta)$ values, depending on the process being looked up.

The second column gives values of $\log_{10}(T_{\pm}(\chi))$ from equation (3) of [Ridgers et al](https://www.sciencedirect.com/science/article/pii/S0021999113008061). This is related to the rate of photon pair-production via the Breit-Wheeler mechanism.

The third column gives values of $\log_{10}(\hat{\Omega}(\eta))$ from equation (4.4) of [Erber](https://journals.aps.org/rmp/abstract/10.1103/RevModPhys.38.626). This is related to the rate of trident pair-production in an external field. Note that Erber refers to $\eta$ as $\Upsilon$, and I think the literature has moved on from even $\eta$ now. I've been using $\eta$ and $\chi$ for consistency with [Ridgers et al](https://www.sciencedirect.com/science/article/pii/S0021999113008061).

Hope this helps,
Stuart
